.github/workflows/build.yml

```diff
@@ -145,6 +149,20 @@ jobs:
         with:
           name: Bicep.LangServer
           path: ./src/vscode-bicep/bicepLanguageServer
+          
+      - name: Run unit tests
+        run: npm run test:unit
+        working-directory: ./src/vscode-bicep
+        
+      - name: Run E2E tests
+        run: xvfb-run -a npm run test:e2e
+        working-directory: ./src/vscode-bicep
+
+      - name: Upload Code Coverage
+        uses: codecov/codecov-action@v1
+        with:
+          flags: typescript
+          directory: ./src/vscode-bicep/coverage
```

Comment:
>coverage [](start = 40, length = 8)

Btw, I'm seeing coverage info at [https://codecov.io/gh/Azure/bicep/pull/686/tree/src/vscode-bicep/src](https://codecov.io/gh/Azure/bicep/pull/686/tree/src/vscode-bicep/src), so this appears to be working. Can you confirm the numbers look reasonable based on our test coverage?

Reply:
Yes, the numbers are reasonable.